packages/runtime/core/src/tsconfig.json

```diff
@@ -1,6 +1,14 @@
 {
   "extends": "../../../../tsconfig-base.json",
   "compilerOptions": {
-    "outDir": "../dist/esm"
-  }
+    "outDir": "../dist/esm",
+    "paths": {
+      "@marlowe.io/adapter/*": ["../../../../adapter/src/*"],
```

Comment:
the adapter part should not be needed with what I added

Reply:
since I have kept `addressBech32` function, this fix is still relevant I believe